libbeat/docs/release-notes/highlights/highlights-7.7.0.asciidoc

```diff
@@ -18,9 +18,131 @@ For a list of bug fixes and other changes, see the {beats}
+monitoring Azure containers.
+
+[float]
+[role="xpack"]
+==== AWS VPCs, Lambdas, and DynamoDB monitoring
+
+In the {metricbeat} AWS module, we've added support for monitoring
+mission-critical services in the Amazon VPC ecosystem:
+
+* The {metricbeat-ref}/metricbeat-metricset-aws-natgateway.html[`natgateway`]
+metricset enables you to monitor NAT gateway services to gain a
+better perspective on how web applications or services are performing.
+* The {metricbeat-ref}/metricbeat-metricset-aws-natgateway.html[`transitgateway`]
+metricset collects metrics sent to CloudWatch by VPC when requests are flowing
+through the gateway. 
+* The {metricbeat-ref}/metricbeat-metricset-aws-vpn.html[`vpn`] metricset
+enables you to monitor VPN tunnels. VPN metric data is automatically sent to
+CloudWatch as it becomes available.
+
+Also new in this release, the
+{metricbeat-ref}/metricbeat-metricset-aws-lambda.html[`lambda`] metricset monitors
+Lambda functions across multiple accounts and regions. The metricset collects
+metrics such as total invocations, errors, duration, throttles, dead-letter queue
+errors, and iterator age for stream-based invocations. You can use these metrics
+to configure alerts to respond to events such as changes in performance and
+error rates. 
+
+We’ve also added the
+{metricbeat-ref}/metricbeat-metricset-aws-dynamodb.html[`dynamodb`] metricset to
+monitor AWS DynamoDB instances. This metricset collects metrics, such as request
+latency, transaction conflicts, provisioned and consumed capacity, and many
+others.    
+
+For Amazon Aurora users, we've enhanced the
+{metricbeat-ref}/metricbeat-metricset-aws-rds.html[`rds`] metricset to collect
+metrics about your Aurora instances.
+
+[float]
+[role="xpack"]
+==== Google Cloud Platform (GCP) Pub/Sub and Load Balancer monitoring
+
+We've enhanced the {metricbeat} Google Cloud Platform module with support
+for monitoring additional services:
+
+* The {metricbeat-ref}/metricbeat-metricset-googlecloud-pubsub.html[`pubsub`]
+metricset connects to the Stackdriver API and collects metrics for topics,
+subscriptions, and snapshots used by a specified account. 
+* The {metricbeat-ref}/metricbeat-metricset-googlecloud-loadbalancing.html[`loadbalancing`]
+metricset captures load balancing performance metrics for HTTP(S), TCP, and UDP
+applications.
+
+[float]
+[role="xpack"]
+==== Pivotal Cloud Foundry (PCF) monitoring
+
+We continue to expand coverage of container platforms by adding support for
+Pivotal Cloud Foundry. 
+
+The new {metricbeat}
+{metricbeat-ref}/metricbeat-module-cloudfoundry.html[Cloudfoundry module]
+connects to the Cloud Foundry API and pulls container, counter, and value
+metrics from it. These metrics are stored in `cloudfoundry.container`,
+`cloudfoundry.counter` and `cloudfoundry.value` metricsets.
+
+In {filebeat}, the new
+{filebeat-ref}/filebeat-input-cloudfoundry.html[`cloudfoundry`] input collects
+http access logs, container logs, and error logs from Cloud Foundry.
+
```

Comment:
Would it be good to link to documentation on deploying metricbeat and filebeat into PCF?

https://www.elastic.co/guide/en/beats/metricbeat/master/running-on-cloudfoundry.html
https://www.elastic.co/guide/en/beats/filebeat/master/running-on-cloudfoundry.html